### ISSUE TITLE
Help: ...https://github.com/qdm12/ddns-updater/blob/master/config.json

### ISSUE THREAD

**Issue by uid65252** [qdm12/ddns-updater#114]
**TLDR**: *Describe your issue in a one liner here*
I try to get running ddns-updater on a raspberry Debian 10
with this command:
docker run -d -p 8000:8000/tcp -v "$(pwd)"/data:/updater/data qmcgaw/ddns-updater
and this config,json file:
https://github.com/qdm12/ddns-updater/blob/master/config.json

In portainer.io log-file it shows:

Running version latest built on 2020-10-31T14:41:32Z (commit 3240bb7)
🔧  Need help? https://github.com/qdm12/ddns-updater/issues/new
💻  Email? quentin.mcgaw@gmail.com
☕  Slack? Join from the Slack button on Github
💸  Help me? https://github.com/sponsors/qdm12
2020-11-09T14:18:23.021Z	ERROR	invalid key format

If i run with command:
docker run -d -p 7000:7000/tcp -v "$(pwd)"/data:/updater/data qmcgaw/ddns-updater {"settings": [{"provider": "",}]}
the same error occured
2020-11-09T15:13:33.087Z	ERROR	invalid key format

1. Is this urgent?

    - [ ] Yes
    - [x] No

2. What DNS service provider(s) are you using?

    - [ ] Cloudflare
    - [ ] DDNSS.de
    - [ ] DonDominio
    - [ ] DNSOMatic
    - [ ] DNSPod
    - [ ] Dreamhost
    - [x] DuckDNS
    - [ ] DynDNS
    - [ ] GoDaddy
    - [ ] Google
    - [ ] He.net
    - [ ] Infomaniak
    - [ ] Namecheap
    - [ ] NoIP

3. What's the version of the program?

    **See the line at the top of your logs**

 4. What are you using to run the container?

    - [x] Docker run
    - [ ] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other: Portainer.io

**Comment by qdm12** [comment]
Can you please share your config.json, and replacing your key/password with `xxxx`? You need to set several fields in the json file as specified in the readme for Duckdns, I'm not sure if this is the case.

**Comment by uid65252** [comment]
thanks for your fast reaction. I made a mistake.
I found my error
Now it is running, thanks for your nice tool

**Comment by qdm12** [comment]
Thanks for letting me know. Enjoy!
